<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgMain</class>
 <widget class="QDialog" name="dlgMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USB-CAN</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>14</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>索引号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>14</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_devType">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>12</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_devIndex">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>12</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>14</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>第几路</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_Chn">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>12</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_connect">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>12</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit_recv">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>531</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtn_clr">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AccCode">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>70</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>验收码:0x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AccMask">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>100</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>屏蔽码:0x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Time0">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>定时器0:0x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>定时器1:0x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Time1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>70</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>滤波方式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_Filter">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>71</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_Mode">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>97</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtn_startCAN">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>启动CAN</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>531</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <zorder>pushBtn_baudHelp</zorder>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>531</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>帧类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_FrameType">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_FrameFormat">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>12</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>帧格式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>12</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>帧ID: 0x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_txdata">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Data">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00 01 02 03 04 05 06 07</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>接收到的数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtn_baudHelp">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率帮助</string>
   </property>
  </widget>
  <zorder>frame_3</zorder>
  <zorder>frame</zorder>
  <zorder>textEdit_recv</zorder>
  <zorder>pushBtn_clr</zorder>
  <zorder>lineEdit_AccCode</zorder>
  <zorder>label_6</zorder>
  <zorder>lineEdit_AccMask</zorder>
  <zorder>label_7</zorder>
  <zorder>lineEdit_Time0</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>lineEdit_Time1</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>cmb_Filter</zorder>
  <zorder>cmb_Mode</zorder>
  <zorder>pushBtn_startCAN</zorder>
  <zorder>frame_4</zorder>
  <zorder>label_16</zorder>
  <zorder>pushBtn_baudHelp</zorder>
 </widget>
 <tabstops>
  <tabstop>pushBtn_connect</tabstop>
  <tabstop>textEdit_recv</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
